Packs/Humio/Integrations/Humio/Humio.yml

```diff
@@ -26,7 +26,7 @@ configuration:
   required: false
 - display: Query to use to fetch incidents
   name: queryParameter
-  defaultvalue: ""
+  defaultvalue: "*"
```

Comment:
Is there a chance that this change can break backwards compatability and return different response then before.
If so you might want to specify it in the release notes.

Reply:
Nope. Quite the opposite. I noticed that queries would fail if the request didn't have a `*` in the parameter. So if anything we are fixing queries that are failing today.

Reply:
In any case; an empty query is equal to a `*` query in Humio.